src/core/WebAppManagerConfig.cpp

```diff
@@ -80,11 +80,11 @@ void WebAppManagerConfig::initConfiguration()
 
 void WebAppManagerConfig::postInitConfiguration()
 {
-    if (access("/var/luna/preferences/debug_system_apps", F_OK) == 0) {
+    if (access("/var/agl-devel/preferences/debug_system_apps", F_OK) == 0) {
```

Comment:
This change should be conditionalized for AGL, so as not to break non-AGL builds.

Reply:
I would be ok with just guarding with IS_AGL for now...

But there is a far better solution imho in the long term. You should make WebAppManagerConfig abstract (or minimal). And move the current implementation to src/webos. Then you need another abstraction for AGL. In this new implementation you would have freedom to implement this with whatever format is preferred in AGL. Then, you also need to refactor a bit PlatformModuleFactoryImpl, to be PlatformModuleFactoryWebOS and provide an alternate for AGL.